MangoPay/Libraries/RestTool.php

```diff
@@ -246,6 +246,13 @@ private function BuildRequest($urlMethod, $pagination, $additionalUrlParams = nu
 
             curl_setopt($this->_curlHandle, CURLOPT_POSTFIELDS, $this->_requestData);
         }
+
+        if (!is_null($this->_root->Config->HostProxy))
```

Comment:
Missing curly braces on both conditions, to stick with the implicit coding standards.